erpnext/selling/page/point_of_sale/point_of_sale.js

```diff
@@ -50,8 +50,8 @@ erpnext.pos.PointOfSale = class PointOfSale {
 				this.set_online_status();
 			},
 			() => this.setup_company(),
-
 			() => this.make_new_invoice(),
+			() => this.set_form_action(),
```

Comment:
Let's hide the print button until there is actually an item in the cart.
![peek 2018-01-09 20-30](https://user-images.githubusercontent.com/818803/34740047-70ed4cca-f57e-11e7-8fe6-8fbebbf40abb.gif)

Reply:
@tundebabzy Thanks for the input. Have made the changes according to the suggestions in new commits.